### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
# Styling
> In this final milestone:
- [x] Add styling to the web app with reference to the Bookstore design in Zeplin.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,277 @@
...
+  display: flex;
+}
+
+.navigation_container {
+  background: #fff;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+.navigation {
+  display: flex;
```
Comment (Mihndim2020) [inline src/index.css]: - [x] In line with `CSS` best practices, keeping our `CSS` dry, it is not nice to use the same rule repetitively, please kindly use a class attribute instead.

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,277 @@
...
+  background: var(--main-acent-color);
+  padding: 5px 12px;
+  width: max-content;
+}
+
+.navigation > ul > li > a {
+  text-decoration: none;
+}
+
+.add-book > * {
```
Comment (Mihndim2020) [inline src/index.css]: -  [x] As it is a requirement for this project and best practice for a front-end dev, please kindly adjust the margin of the `Addbook Section` to match the design.

***Your implementation***
yourI

***The design***
theD

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Your project is running as expected

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (Chimwemwe-127001) [reply src/index.css]: Done

File: src/index.css (hunk 2 above)
Comment (Chimwemwe-127001) [reply src/index.css]: Done

**Comment by Chimwemwe-127001** [comment]
- Hello @Mihndim2020. All changes requested have been actioned. Hopefully, my PR gets approved this time
- Happy coding!!! :computer: :grinning:

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @Chimwemwe-127001 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**